docs/fsharp/tutorials/fsharp-interactive/index.md

```diff
@@ -88,6 +79,36 @@ test
 90
 ```
 
+## Package Management in F# Interactive
+
+[!NOTE] Package management is available as a preview feature in versions of `dotnet fsi` shipped in the `3.1.300` and greater versions of the .Net Core SDK, as well as all `5.*` versions of the .Net SDK. To enable it in this preview release, run `dotnet fsi` with the `--langversion:preview` argument.
```

Comment:
```suggestion
[!NOTE] Package management is available as a preview feature in versions of `dotnet fsi` shipped in the `3.1.300` and greater versions of the .NET SDK, as well as all `5.*` versions of the .NET SDK. To enable it in this preview release, run `dotnet fsi` with the `--langversion:preview` argument.
```

The versions are correct, just making the names the same (even though your statement is technically correct).